sample_rules/simple/simple0.yaml

```diff
@@ -0,0 +1,7 @@
+# YAML
+# Goal: Turn on back light when in reverse
+
+condition:
+ - state.transmission_gear == 'reverse'
```

Comment:
What's up with state. here? And I believe the samples we have seen say transmission.gear, isn't it?